packages/block-editor/src/components/color-palette/control.js

```diff
@@ -4,15 +4,13 @@
 import ColorGradientControl from '../colors-gradients/control';
 
 export default function ColorPaletteControl( {
-	colors,
-	disableCustomColors,
-	label,
 	onChange,
 	value,
+	...otherProps
 } ) {
 	return (
 		<ColorGradientControl
-			{ ...{ colors, disableCustomColors, label } }
+			{ ...otherProps }
```

Comment:
Is there a standard for this? `other` seems a little vague. If these props have a specific use case that can be semantically described then perhaps we should make the prop name clearer as this will make the component easier to consume.

Reply:
yes, it makes sense. However, [this changed](f5c7d7f979a83afa94d3f72391c71b2b9f0c2df6) was introduced in [another PR](https://github.com/WordPress/gutenberg/pull/19133). I just rebased this one in order to make it work. Rebasing again...